app/dashboard/templates/shared/menu.html

```diff
@@ -0,0 +1,307 @@
+        <div class="gc-megamenu-content">
+          <div class="row">
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/bount' in request.path or '/explorer' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url "funder_bounties" %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/bounties_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Bounties" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Development resources at your finger tips" %}</p>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'new_bounty' %}">{% trans "Fund Issue" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'explorer' %}">{% trans "Explore Bounties" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'quickstart' %}">{% trans "Funder Quickstart" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'funder_bounties' %}">{% trans "About Bounties" %}</a>
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/hackathon' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'get_hackathons' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/hackathons_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Hackathons" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Build with the coolest web3 projects" %}</p>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'get_hackathons' %}">{% trans "Explore Hackathons" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="http://hackathons.gitcoin.co/contact" target="_blank">{% trans "Get Involved" %}</a>
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/grants' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'grants:grants' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/grants_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Grants" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Recurring Funding for open source" %}</p>
```

Comment:
Capitalize "open source"? what do you think? I notice the capitalization is a bit different on some of the items